planner/core/point_get_plan.go

```diff
@@ -877,12 +878,13 @@ func tryPointGetPlan(ctx sessionctx.Context, selStmt *ast.SelectStmt, check bool
 		return nil
 	}
 
+	isCacheTable := tbl.TableCacheStatusType == model.TableCacheStatusEnable
 	var partitionInfo *model.PartitionDefinition
 	var pos int
 	if pi != nil {
 		partitionInfo, pos, isTableDual = getPartitionInfo(ctx, tbl, pairs)
 		if isTableDual {
-			p := newPointGetPlan(ctx, tblName.Schema.O, schema, tbl, names)
+			p := newPointGetPlan(ctx, tblName.Schema.O, schema, tbl, names, isCacheTable)
```

Comment:
Since `isCacheTable  == tbl.TableCacheStatusType == model.TableCacheStatusEnable` . It is not necessary to add it to the plan and you can just use `pointGetPlan.TblInfo` instead